mocks/openedx/core/lib/api/authentication/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""
+Mocks for openedx.core.lib.api.authentication.
+"""
+
+from rest_framework.authentication import BaseAuthentication
+
+
+class OAuth2Authentication(BaseAuthentication):
+    """
+    Creating temperary class cause things outside of edx-platform need OAuth2Authentication.
+    This will be removed when repos outside edx-platform import BearerAuthentiction instead.
+
+    Note (Omar): Could be removed in the Koa release. Nothing to do for now, but if it breaks, we need
+                 to switch to BearerAuthentiction.
+    """
+
+    def authenticate(self, request):
+        """
+        Fake authentication method that doesn't authenticate.
+        """
+        return
```

Comment:
From the [DRF docs](https://www.django-rest-framework.org/api-guide/authentication/#custom-authentication), looks like this method should return a tuple of two values, `(User, auth)`, where `auth` can be `None`

Reply:
Thanks for checking. After looking at the docs again, I think `None` is fine:

> **Custom authentication**
> To implement a custom authentication scheme, subclass BaseAuthentication and override the .authenticate(self, request) method. The method should return a two-tuple of (user, auth) if authentication succeeds, **or None otherwise.**